ValidationToolkit.xcodeproj/project.pbxproj

```diff
@@ -1156,7 +1141,7 @@
 				PRODUCT_NAME = ValidationToolkit;
 				SDKROOT = macosx;
 				SKIP_INSTALL = YES;
-				SWIFT_VERSION = 4.0;
+				SWIFT_VERSION = 5.0;
```

Comment:
Please migrate the `tvOS` and `watchOS` targets to Swift 5. I think it's only necessary to update the value in the `Build Settings`.

<img width="826" alt="Screenshot 2019-04-17 at 19 34 17" src="https://user-images.githubusercontent.com/851201/56305044-133fd200-6148-11e9-8bbf-8add7ff1b24a.png">

Reply:
👍